docs/nodes/nodes.md

```diff
@@ -89,7 +89,7 @@ export default class MathNode extends Node<Inputs, Outputs> {
     public inputs = {
         number1: new NumberInterface("Number", 1),
         number2: new NumberInterface("Number", 2),
-        operation: new SelectInterface("Operation", "Add").setPort(false),
+        operation: new SelectInterface("Operation", "Add", , ["Add", "Subtract"]).setPort(false),
```

Comment:
```suggestion
        operation: new SelectInterface("Operation", "Add", ["Add", "Subtract"]).setPort(false),
```
The constructor with four parameters was only used for v1, in v2 there are only three parameters: https://v2.baklava.tech/api/classes/_baklavajs_renderer_vue.SelectInterface.html